app/workers/education_form/education_facility.rb

```diff
@@ -68,8 +68,8 @@ def self.facility_for(region:)
     def self.region_for(model)
       record = model.open_struct_form
       address = routing_address(record, form_type: model.form_type)
-      # special case Philippines
-      return :western if address&.country == 'PHL' || model.form_type == '0993'
+
+      check_special_cases(address, model)
```

Comment:
this isn't going to return here

Reply:
Good catch. Thanks. Its been updated now